earth_enterprise/src/third_party/gdal/README.txt

```diff
@@ -35,3 +35,11 @@ on a SLES 9 box (compatible with SLES 9,10, RHEL 4,5 and Ubuntu 6,8)
 
 To Build, you will need to build on a SLES 9 box using the scripts found in
 //depot/googleclient/geo/fusion/RPMS/build_scripts
+
+To build GDAL with MrSID Raster Format, MrSID library needs to be installed on your platform:
+1- Get MrSID Decoding SDK library
+2- Copy MrSDK library into a folder. Ex: /opt
```

Comment:
Would it be better to put this in /usr/local/lib?

Reply:
I contemplated that idea, but the way GDAL expect MrSid library is a folder where there is a 'lib' sub-folder and 'include' sub-folder. Using 'usr/local/lib' might indicate that the library is stored there while the headers are store in 'usr/local/include'.